dali/ft/daft.cpp

```diff
@@ -78,6 +78,7 @@ void CDistributedFileSystem::exportFile(IDistributedFile * from, IFileDescriptor
     sprayer->setAbort(abort);
     sprayer->setPartFilter(filter);
     sprayer->setSource(from);
+    sprayer->checkTarget(to);
```

Comment:
As this CDistributedFileSystem::exportFile() is the entry point to despray and this is the only place that needs to call checkTarget().
I'd have probably put the method to validate exports in CDistributedFileSystem and check above, before sprayer object is created.